functions/Grant-YouTubeOauth.ps1

```diff
@@ -43,16 +48,10 @@ function Grant-YoutubeOauth {
     'https://www.googleapis.com/auth/youtube.force-ssl'
     'https://www.googleapis.com/auth/youtube.readonly'
   )
-  $Uri = 'https://accounts.google.com/o/oauth2/v2/auth?include_granted_scopes=true&response_type=token&client_id={0}&redirect_uri={1}&scope={3}&state={2}' -f $Client.client_id, $RedirectUri, (New-Guid).Guid, ($ScopeList -join ' ')
+  $scopesURI = [System.Web.HttpUtility]::UrlEncode($ScopeList -join ' ')
+  $Uri = 'https://accounts.google.com/o/oauth2/v2/auth?include_granted_scopes=true&response_type=token&client_id={0}&redirect_uri={1}&scope={2}&state={2}' -f $Client.client_id, $RedirectUri, ($scopesURI), (New-Guid).Guid
 
```

Comment:
Does this fix something? No need for the parentheses around `($scopesURI)`.

Reply:
without encoding, Edge on Windows opens each scope "URL" in a separate tab.